src/Utils.ts

```diff
@@ -10,9 +10,7 @@ function sanitizeUrls(urls: string[]) {
     const sanitized: string[] = [];
 
     for (let i=0, l=urls.length; i<l; ++i) {
-        const urlAr = urls[i].split('?');
-        const query = urlAr.length === 2 && urlAr[1] !== '' ? '?'+urlAr[1] : '';
-        let url = urlAr[0];
+        let url = urls[i].split('?')[0];
```

Comment:
What does this do if there's no `?` in the URL?

Reply:
it returns urls[i]

as pointed out by constructor-s, split always returns an array
it comes in handy in such cases

Reply:
Oh yes i remember that comment now. Cool. Merging into master.